libbeat/processors/add_process_metadata/_meta/fields.yml

```diff
@@ -0,0 +1,12 @@
+- key: process
```

Comment:
It seems this file is currently ignored, I'm wondering about other fields.yml in the processors?

Reply:
I first "thought" it's not working because I already had the mindset something in the fields.go in libbeat must change. But it works but the fields.go in all Beats changes. False alarm on my side.